.github/workflows/release.yml

```diff
@@ -32,18 +32,23 @@ jobs:
   manylinux-release-wheel:
     name: Build release wheels for manylinux2010
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ['3.5', '3.6', '3.7']
     steps:
       - if: github.event_name == 'schedule'
         run: echo "::set-env name=NIGHTLY_FLAG::--nightly"
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
         with:
-          python-version: 3.5
+          python-version: ${{ matrix.python-version }}
       - name: Build manylinux2010 wheels
+        env:
+          PY_VERSION: ${{ matrix.python-version }}
         run: |
```

Comment:
Can we add an if statement similar to the others:
```
 if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
```
Probably best to do this after we confirm that we're correctly building 3.5/3.6/3.7